tests/kernel/fifo/fifo_timeout/src/main.c

```diff
@@ -214,8 +216,10 @@ static int test_multiple_threads_get_data(struct timeout_order_data *test_data,
 		TC_PRINT(" thread (q order: %d, t/o: %d, fifo %p)\n",
 				data->q_order, data->timeout, data->fifo);
 	} else {
-		TC_ERROR(" *** thread %d woke up, expected %d\n",
+		if (data) {
+			TC_ERROR(" *** thread %d woke up, expected %d\n",
 					data->timeout_order, ii);
```

Comment:
I think the expectation here is data->q_order and not data->timeout_order, let me fix that. Correct me if I am wrong.